### PR TITLE
feat: [SRM-15105]: Added Deployment Analysis Event Card

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

image

Screenshot 2023-07-17 at 9 31 17 AM

Screenshot 2023-07-17 at 9 31 01 AM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/utils/CommonUtils.ts [hunk 1]
```diff
@@ -345,6 +346,8 @@ export const getDetailsLabel = (key: string, getString: UseStringsReturn['getStr
       return getString('common.updatedBy')
     case 'externalLinkToEntity':
       return getString('cv.changesPage.externalLink')
+    case 'deploymentImpactAnalysis':
```
Comment (harshilbits) [inline src/modules/85-cv/utils/CommonUtils.ts:349]: pls use the existing enum

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx [hunk 2]
```diff
@@ -52,14 +62,34 @@ export default function VerificationStatusCard({
       backgroundColor = Color.GREEN_350
       icon = 'deployment-success-new'
       break
+    case 'COMPLETED':
```
Comment (harshilbits) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/VerificationStatusCard/VerificationStatusCard.tsx:65]: can we use the existing enum here or define a new enum ?

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/CVProgressBar/CVProgressBar.tsx [hunk 3]
```diff
@@ -27,11 +27,15 @@ function statusToIntent(status: CVProgressBarProps['status']): string {
     case 'VERIFICATION_FAILED':
       return Classes.INTENT_DANGER
     case 'IN_PROGRESS':
+    case 'RUNNING':
       return Classes.INTENT_PRIMARY
     case 'NOT_STARTED':
       return ''
+    case 'COMPLETED':
```
Comment (harshilbits) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentProgressAndNodes/components/CVProgressBar/CVProgressBar.tsx:34]: can we use the existing enum here or define a new enum ?

**Review by harshilbits — DISMISSED** [review]

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx [hunk 4]
```diff
@@ -37,20 +36,18 @@ export default function ColumChartWithStartAndStopEventMarker({
   return (
     <div style={{ position: 'absolute', top: 20, display: 'flex' }}>
       <ColumnChartEventMarker
-        data-testId="startMarker"
         columnHeight={columnHeightDerived}
         customMarker={DeploymentImage}
         leftOffset={startMarkerPosition}
-        markerColor={Color.GREEN_400}
+        markerColor={'var(--green-400)'}
```
Comment (karansaraswat19) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx:42]: Why did we stop using Color from design-system?

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx [hunk 5]
```diff
@@ -37,20 +36,18 @@ export default function ColumChartWithStartAndStopEventMarker({
...
       />
       <ColumnChartOverlay isStopped={isStopped} startMarkerPosition={startMarkerPosition} width={shadowWidth} />
       {deployedOrStopMarkerPosition ? (
         <ColumnChartEventMarker
-          data-testId="deployedOrStopMarker"
           columnHeight={columnHeightDerived}
           customMarker={isStopped ? StopImage : SuccessImage}
           leftOffset={deployedOrStopMarkerPosition}
-          markerColor={isStopped ? Color.GREY_200 : Color.GREEN_400}
+          markerColor={isStopped ? 'var(--gray-200)' : 'var(--green-400)'}
```
Comment (karansaraswat19) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx:50]: Shouldn't be it var(--grey-200)?

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx (hunk 4 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx:42]: This is a new component we added in previous PR, that PR has the component only , not integration 
on integrating Color.GREEN_400  was not working , so used variable

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx (hunk 5 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx:50]: This is a new component we added in previous PR, that PR has the component only , not integration 
on integrating Color.GREEN_400  was not working , so used variable

Comment (karansaraswat19) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeCard/components/SLOAndErrorBudget/ColumChartWithStartAndStopEventMarker/ColumChartWithStartAndStopEventMarker.tsx:50]: I am talking about typo here

**Review by karansaraswat19 — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Pull request merged**